src/main/java/dk/cngroup/lentils/repository/HintTakenRepository.java

```diff
@@ -22,6 +22,11 @@
             nativeQuery = true)
     List<HintTaken> findAllByTeamAndCypher(Long teamId, Long cypherId);
 
+    @Query(value = "SELECT ht FROM HintTaken ht " +
+            "LEFT JOIN Hint h ON ht.hint.hintId = h.hintId " +
+            "WHERE h.cypher.cypherId=?1 ORDER BY h.value ASC")
+    List<HintTaken> findAllByCypher(Long cypherId);
```

Comment:
Tohle by šlo myslím taky přes JPQL `findAllByHintCypherCypherId`.
Ostatně to nad tím, co jsem kdysi dělal já(:rofl: ) by šlo asi také přes JPQL, tak jestli můžeš `findAllByTeamTeamIdAndHintCypherCypherId` :innocent:

Reply:
super, upraveno ;)